src/components/Table/TableBody/TableBodyRow/TableBodyRow.jsx

```diff
@@ -177,6 +177,17 @@ const StyledTableExpandRow = styled(({ hasRowSelection, ...props }) => (
   <TableExpandRow {...props} />
 ))`
   &&& {
+    ${
+      // if single nested hierarchy AND there are children rows (meaning this is a parent),
```

Comment:
I think that this comment is superfluous. If you would extract `props['data-child-count'] > 0` into a `isParent` const then the code would read exactly like your comment.

Reply:
Great point. Looks like it was merged, but I should have another fix coming in soon where I can add this in